modules/ensemble/lib/widget/helpers/controllers.dart

```diff
@@ -468,6 +478,8 @@ abstract class EnsembleWidgetController extends EnsembleController
       'flexMode': (value) => flexMode = FlexMode.values.from(value),
       'flex': (value) => flex = Utils.optionalInt(value, min: 1),
       'visible': (value) => visible = Utils.getBool(value, fallback: true),
+      'opacity': (value) =>
+          opacity = Utils.optionalDouble(value, min: 0, max: 100),
```

Comment:
huh? where is 100 coming from? why not use the getValidOpacity function instead of optionalDouble?